src/Microsoft.Health.Fhir.Liquid.Converter/Utilities/Hl7v2DataUtility.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Utilities
+{
+    public static class Hl7v2DataUtility
+    {
+        private static readonly string[] SegmentSeparators = { "\r\n", "\r", "\n" };
+
+        public static string[] SplitSegment(string message)
```

Comment:
The function is more about `SplitIntoSegments` or `SplitMessageToSegments` instead of `SplitSegment`?

Reply:
OK. I can rename it as `SplitMessageToSegments`.